src/api/app/models/user.rb

```diff
@@ -768,20 +768,21 @@ def outgoing_requests(search = nil)
 
   # list of all requests
   def requests(search = nil)
-    bs_requests_table = BsRequest.arel_table
-    bs_actions_table = BsRequestAction.arel_table
-    reviews_table = Review.arel_table
-
-    result =
-      BsRequest.in_states(VALID_REQUEST_STATES).with_actions_and_reviews
-               .where(bs_requests_table[:creator].eq(login)
-                 .or(bs_actions_table[:target_project_id].in(involved_projects.pluck(:id)))
-                 .or(bs_actions_table[:target_package_id].in(involved_packages.pluck(:id)))
-                 .or(reviews_table[:state].eq(:new)
-                     .and(reviews_table[:user_id].eq(id)
-                     .or(reviews_table[:project_id].in(involved_projects.pluck(:id)))
-                     .or(reviews_table[:package_id].in(involved_packages.pluck(:id)))
-                     .or(reviews_table[:group_id].in(groups.pluck(:id))))))
+    projects = involved_projects.pluck(:id)
+    packages = involved_packages.pluck(:id)
```

Comment:
Theses variables store ids in them so they should be called `project_ids` and `package_ids`